app/api/attendees.py

```diff
@@ -31,6 +31,10 @@ def before_post(self, args, kwargs, data):
         """
         require_relationship(['ticket', 'event'], data)
 
+        ticket = db.session.query(Ticket).filter_by(id=int(data['ticket'])).first()
+        if ticket.event_id != int(data['event']):
```

Comment:
Ticket can be `None`

Reply:
Ok, I've added null check for `ticket` and both event and ticket relationships as well.